bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -250,7 +250,7 @@ impl EventTimelineItem {
                 remote_event_item
                     .reactions()
                     .iter()
-                    .map(|(k, v)| Reaction { key: k.to_owned(), count: v.count.into() })
```

Comment:
It's still a better idea to `.try_into().unwrap()` to avoid surprises.
Well, here it's a `usize` to `u64`, so it's unlikely to be broken, but still :-).